.github/workflows/ci-release.yml

```
@@ -223,24 +294,22 @@ jobs:
+      if: env.KLOGG_HAS_SENTRY_TOKEN == 'true'
+      uses: mathrix-education/setup-sentry-cli@ff4fff773f5e628fa214ebd19e46fb7289454ee2 # v3.0.0
       with:
         token: ${{ secrets.SENTRY_TOKEN }}
         organization: anton-filimonov
         project: klogg
 
     - name: Create Sentry release
-      if: env.SENTRY_TOKEN != ''
+      if: env.KLOGG_HAS_SENTRY_TOKEN == 'true'
       shell: sh
       run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:1:25: Double quote to prevent globbing and word splitting

(shellcheck)